app/contestants/list_of_contestants.py

```diff
@@ -0,0 +1,88 @@
+class Contestants:

```

Comment:
There is no need to create a separate class to only store a variable. Make `KNIGHTS` a separate variable and import it directly where needed.